src/component/common/SidebarModal/SidebarModal.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode } from 'react';
+import { Modal, Backdrop, Slide } from '@material-ui/core';
+import Fade from '@material-ui/core/Fade';
+import { useStyles } from 'component/common/SidebarModal/SidebarModal.styles';
+
+interface ISidebarModalProps {
+    open: boolean;
+    onClose: () => void;
+    label: string;
+    children: ReactNode;
+}
+
+const TRANSITION_DURATION = 250;
+
+export const SidebarModal = ({
+    open,
+    onClose,
+    label,
+    children,
+}: ISidebarModalProps) => {
+    const styles = useStyles();
+
+    return (
+        <Modal
+            open={open}
+            onClose={onClose}
+            closeAfterTransition
+            aria-label={label}
+            BackdropComponent={Backdrop}
+            BackdropProps={{ timeout: TRANSITION_DURATION }}
+        >
+            <Fade timeout={TRANSITION_DURATION} in={open}>
+                <div>
+                    <Slide
```

Comment:
In addition, when clicking edit strategy it fades in instead of sliding. I would prefer the behaviors to be the same.

Reply:
The slide-out is a bit of work since we control the modal state through routing and path parameters.

I've removed the slide-in at least, so that the animation is consistent.